README.md

```diff
@@ -6,10 +6,71 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](LICENSE-MIT)
 [![License: Apache 2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE-APACHE)
 
-Instance Distance is a fast pure-Rust implementation of the
-[Hierarchical Navigable Small Worlds paper][paper] by Malkov and Yashunin
-for finding approximate nearest neighbors. This implementation will power the
+Instance Distance is a fast pure-Rust implementation of the [Hierarchical
+Navigable Small Worlds paper][paper] by Malkov and Yashunin for finding
+approximate nearest neighbors. This implementation powers the
 [InstantDomainSearch.com][ids] backend services used for word vector indexing.
 
+## What it does
+
+Instant Distance is an implementation of a fast approximate nearest neighbor
+search algorithm. The algorithm is used to find the closest point(s) to a given
+point in a set.
+
+## Using the library
+
+### Rust
+
+```toml
+[dependencies]
+instant-segment = "0.5.0"
+```
+
+## Example
+
+```rust
+use instant_distance::{Builder, Search};
+
+fn main() {
+    //
```

Comment:
Probably should just remove this?

Reply:
It is redundant, but I do also like having an example right in the readme for clarity.